lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -36,6 +36,47 @@ static function () use ( $style ) {
 	);
 }
 
+/**
+ * Fetches, processes and compiles stored core styles, then combines and renders them to the page.
+ * Styles are stored via the style engine API. See: packages/style-engine/README.md
+ */
+function gutenberg_enqueue_stored_styles() {
```

Comment:
Just checking, the idea with using a generic name like `wp_enqueue_stored_styles` rather than being tied to the style engine, is that we're not exposing any style engine behaviour here? I think that sounds good to me, in that this is about enqueuing stored styles in general, (and deals with essential styles for WordPress) rather than being specific to the style engine.

Reply:
Another tricky naming issue 😄 

"Stored" indirectly implies the style engine, but yeah the motivation was to expand this function later to apply to all stores. See [comment above](https://github.com/WordPress/gutenberg/pull/42880#discussion_r935208342).

If that's true, it will probably require a strategy to subsume/integrate `gutenberg_enqueue_global_styles()` when we get to global styles. ?

Reply:
>  is that we're not exposing any style engine behaviour here?

Only pulling from style engine stores. Do you think we should be more specific in the naming? 🤔

Reply:
> Do you think we should be more specific in the naming? 🤔

Naming's hard! For anything that's specifically style engine related (rather than the style engine being an implementation detail), I like the idea of including `style_engine` in the public function name so that it's all grouped together. So I'd probably lean toward `wp_style_engine_enqueue_stored_styles` as the function name, but I don't feel at all strongly about it if you prefer the shorter name 🙂

Reply:
This sounds infinitely reasonable.

I don't really have a preference either 🙉 

I suppose a weak argument to keep it generic is that `wp_style_engine_enqueue_stored_styles` might communicate to some that it's a public function of the style engine package.

Reply:
> I suppose a weak argument to keep it generic is that wp_style_engine_enqueue_stored_styles might communicate to some that it's a public function of the style engine package.

Good point! Since neither of us feels strongly about it, I think that nudges us more toward keeping the shorter title — after all, it's a function defined outside of the style engine package.